CITATION.cff

```diff
@@ -56,6 +55,6 @@ keywords:
   - bioimaging
   - ndimensional-arrays
 license: BSD-3-Clause
-commit: a4d43a2a8cd8d4abce00e475f9f1699d451eba9f
-version: 0.1.0
-date-released: '2024-02-14'
+commit: cea2e59361548db14fc400eafbfa97446dfdd33e
```

Comment:
Should this match the [release](https://github.com/czbiohub-sf/iohub/releases/tag/v0.2.0) hash (`dc704aa5506a84fdb2b538834ac181b691e70766`)?

Reply:
Good catch...
```suggestion
commit: dc704aa5506a84fdb2b538834ac181b691e70766
```